src/lib/components/sparkline/SparkLine.component.js

```diff
@@ -0,0 +1,129 @@
+//@flow
+import React from "react";
+import VegaChart from "../vegachart/VegaChart.component.js";
+import { lighten } from "polished";
+type Props = {
+  id: string,
+  title?: string,
+  width?: number,
+  height?: number,
+  data: Array<Object>,
+  xAxis: Object,
+  yAxis: Object,
+  row: Object,
+  color?: string,
+  maxRuleColor?: string,
+  minRuleColor?: string
+};
+
+function SparkLine({
+  id,
+  title,
+  width = 300,
+  height = 80,
+  data,
+  xAxis,
+  yAxis,
+  row,
+  color,
+  maxRuleColor = "#F1B434",
+  minRuleColor = "#006F62"
+}: Props) {
+  const maxRuleColorLight = lighten(0.5, maxRuleColor).toString();
+  const minRuleColorLight = lighten(0.5, minRuleColor).toString();
```

Comment:
we should not use `lighten` since  if `maxRuleColor` is already `light`, maxRuleColorLight will be white

Reply:
ok. So I will remove `maxRuleColorLight` and `minRuleColorLight` just use opacity.